index.html

```diff
@@ -74,259 +73,263 @@ <h2 class="h2 color-dark">
-								<img src="images/icons/github.svg" alt="Github icon" />
-							</a>
-							<a href="https://www.linkedin.com/" rel="noopener noreferrer" target="_blank">
-								<img src="images/icons/linkedin.svg" alt="Linkedin icon" />
-							</a>
-							<a href="http://www.facebook.com/" target="_blank" rel="noopener noreferrer">
-								<img src="images/icons/hand.svg" alt="Hand icon" />
-							</a>
-							<a href="https://twitter.com/" rel="noopener noreferrer" target="_blank">
-								<img src="images/icons/twitter.svg" alt="Twitter icon" />
-							</a>
-							<a href="http://www.youtube.com/" rel="noopener noreferrer" target="_blank">
-								<img src="images/icons/m.svg" alt="M icon" />
-							</a>
-						</div>
-						<div class="about__actions">
-							<a href="https://github.com/Rysth" class="button button__primary">Get my resume</a>
-						</div>
-					</header>
-					<div class="about__categories">
-						<div class="about__category">
-							<header class="about__header flex-row justify-between align-center">
-								<h3 class="about__subtitle">Languages</h3>
-								<img src="images/icons/arrow-down.svg" alt="Arrow down icon" />
-							</header>
-							<ul class="about__list grid">
-								<li class="about__item">
-									<img src="images/categories/languages/javascript.png" alt="javascript png image" />
-									JavaScript
-								</li>
-								<li class="about__item">
-									<img src="images/categories/languages/html.png" alt="javascript png image" />
-									HTML
-								</li>
-								<li class="about__item">
-									<img src="images/categories/languages/css.png" alt="javascript png image" />
-									CSS
-								</li>
-							</ul>
-						</div>
-						<div class="about__category">
-							<header class="about__header flex-row justify-between align-center">
-								<h3 class="about__subtitle">Frameworks</h3>
-								<img src="images/icons/arrow-right.svg" alt="Arrow right icon" />
-							</header>
-						</div>
-						<hr />
-						<div class="about__category">
-							<header class="about__header flex-row justify-between align-center">
-								<h3 class="about__subtitle">Skills</h3>
-								<img src="images/icons/arrow-right.svg" alt="Arrow right icon" />
-							</header>
+				<div class="container">
+					<div class="about padding-5 grid grid-two-columns">
+						<header class="about__header flex-column justify-center">
+							<h2 class="h2 color-dark">About Myself</h2>
+							<p class="color-info">
+								Hello I'm a software developer! I can help you build a product, feature or website. Look
+								through some of my work and experience! If you like what you see and have a project you need
+								coded, don't hesitate to contact me.
+							</p>
+							<a
+								href="https://github.com/Rysth"
+								class="primary__link"
+								target="_blank"
+								rel="noopener noreferrer"
+								>Let's Connect</a
+							>
```

Comment:
- [ ] I noticed that **your About section elements are centered and have the same `width`**, but according to the [template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=39-122), the elements should have a different `width`. As a hint, **the left element should take up 40% of the page's width, while the right element should occupy the remaining 60%**. Adjusting the layout in this way **will bring it closer to the desired design.** 😃

- [ ] I noticed that the position **of your language icons and names in the contact section is currently centered.** However, in the design, they are supposed **to be aligned to the left.** Could you please take a moment to review and adjust their positioning accordingly? **It will help maintain consistency with the intended design.** 🎯

   ### Your design:
   ![image](https://github.com/Rysth/Portfolio-Project/assets/107952724/946cd62d-3840-4b23-99e5-ea82056a60d0)


   ### Figma design:
   ![image](https://github.com/MustafaSamiShirzad/portfoliowebsite2/assets/107952724/73937fce-ef09-41ae-ad88-0751761743b8)

Reply:
Thanks! I did changes to the About Section
Using a class. 40% & 60%.